applications/content-drafting.md

```diff
@@ -0,0 +1,29 @@
+Machine translation allows to quickly draft content in natural language. Applications:
+
+# Mock-up localization
+Raw machine-translated text is a good mock-up to verify how the target language appears in localized content such as:
+* user interface,
+* images and diagrams.
+
+MT produces natural-language content that shall not be verified for accuracy at this step, but presents such features as target alphabet and typical word/sentence length. 
```

Comment:
Instead of `MT`, let's use its full format: `Machine translation`, following our [style guide](https://machinetranslate.org/style#acronyms-and-abbreviations).

Reply:
I expanded to full format the first occurence in every section, I leave MT in further occurences. Please check if I also need to expand those.

Reply:
If you put `Machine translation (MT)` in the first occurrence, then you can just use MT later. :)